base/cvd/cuttlefish/host/libs/web/android_build_api.h

```diff
@@ -37,37 +38,7 @@ namespace cuttlefish {
 inline constexpr char kAndroidBuildServiceUrl[] =
     "https://www.googleapis.com/android/internal/build/v3";
 
-struct DeviceBuild {
-  DeviceBuild(std::string id, std::string target,
-              std::optional<std::string> filepath);
-
-  std::string id;
-  std::string target;
-  std::string product;
-  std::optional<std::string> filepath;
-};
-
-std::ostream& operator<<(std::ostream&, const DeviceBuild&);
-
-struct DirectoryBuild {
-  // TODO(schuffelen): Support local builds other than "eng"
-  DirectoryBuild(std::vector<std::string> paths, std::string target,
-                 std::optional<std::string> filepath);
-
-  std::vector<std::string> paths;
-  std::string target;
-  std::string id;
-  std::string product;
-  std::optional<std::string> filepath;
-};
-
-std::ostream& operator<<(std::ostream&, const DirectoryBuild&);
-
-using Build = std::variant<DeviceBuild, DirectoryBuild>;
-
-std::ostream& operator<<(std::ostream&, const Build&);
-
-class BuildApi {
+class BuildApi : public IBuildApi {
```

Comment:
I'd prefer the interface to be named `BuildApi` and the base class `AndroidBuildApi`.